sklearnex/ensemble/tests/test_random_forest.py

```diff
@@ -36,4 +36,4 @@ def test_sklearnex_import_rf_regression():
                            random_state=0, shuffle=False)
     rf = RandomForestRegressor(max_depth=2, random_state=0).fit(X, y)
     assert 'daal4py' in rf.__module__
-    assert_allclose([-6.66], rf.predict([[0, 0, 0, 0]]), atol=1e-2)
+    assert_allclose([-6.97], rf.predict([[0, 0, 0, 0]]), atol=1e-2)

```

Comment:
what is the value in stock?

Reply:
In stock Sklearn it's near -8,32; new implementation is close